### PR TITLE
Adds a new drawing tool for curves

### PR DESCRIPTION
**Pull request by CKrawczyk**
Here is a new curve drawing tool I have been working on for one of the Portsmouth based projects (galaxy zoo 3D).

### CODE REVIEW

File: app/classifier/drawing-tools/curve.cjsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    initCoords: null
+
+    defaultValues: ({x, y}) ->
+      points: []
+      closed: false
+
+    initStart: (newPoint, mark) ->
+      if mark.points.length > 0
+        lastEnd = mark.points[-1..][0]
+        cp = @toImageFrame(lastEnd, newPoint, DEFAULT_CURVE)
```
Comment (brian-c) [inline app/classifier/drawing-tools/curve.cjsx]: I'm not sure what `cp` stands for. Let's try not to abbreviate anything, the more descriptive the better.

File: app/classifier/drawing-tools/curve.cjsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        mark.points.push cp
+      ###
```
Comment (brian-c) [inline app/classifier/drawing-tools/curve.cjsx]: Is this comment old code? Let's clean it out.

File: app/classifier/drawing-tools/curve.cjsx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    guideWidth = GUIDE_WIDTH / averageScale
+
+    firstPoint = points[0]
+    secondPoint = points[1]
+    secondPoint ?=
+      x: firstPoint.x + (finisherRadius * 2)
+      y: firstPoint.y - (finisherRadius * 2)
+    lastPoint = points[points.length - 1]
+
+    deleteButtonPosition =
```
Comment (brian-c) [inline app/classifier/drawing-tools/curve.cjsx]: This ends up on the guideline. How hard would it be to put it on the curve?

**Comment by brian-c** [comment]
Made some comments inline.

This is a super cool tool, but it's more of a polycurve than a curve. Could we rename it in case there's a single-curve tool in the future?

File: app/classifier/drawing-tools/curve.cjsx (hunk 1 above)
Comment (CKrawczyk) [inline app/classifier/drawing-tools/curve.cjsx]: Changed `cp` to `controlPoint` in the code.

File: app/classifier/drawing-tools/curve.cjsx (hunk 2 above)
Comment (CKrawczyk) [inline app/classifier/drawing-tools/curve.cjsx]: Yep, old code I forgot to remove.

File: app/classifier/drawing-tools/curve.cjsx (hunk 3 above)
Comment (CKrawczyk) [inline app/classifier/drawing-tools/curve.cjsx]: Fixed this.

**Comment by CKrawczyk** [comment]
I changed the name of the tool to `bezier` since that is the type of curve being used.

**Comment by brian-c** [comment]
Super cool tool, thanks!

**Pull request merged**